# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../fltk/make.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- fltk-2.0.x-r5479/Makefile	2006-07-11 09:32:27.000000000 +0000
+++ fltk-2.0.x-r5479-fixed/Makefile	2006-10-05 11:53:39.000000000 +0000
@@ -25,7 +25,7 @@
 
 include makeinclude
 
-DIRS = src $(LOCALIMAGES) images OpenGL fluid glut test
+DIRS = src $(LOCALIMAGES) images OpenGL fluid
 
 all: makeinclude
 	for dir in $(DIRS); do\
